src/main/kotlin/io/snyk/plugin/ui/jcef/Utils.kt

```diff
@@ -31,4 +29,18 @@ object JCEFUtils {
 
         return jbCefBrowser.component
     }
+
+    private fun getBrowser(): Pair<JBCefClient, JBCefBrowser> {
+        if (jbCefPair != null) {
+            return jbCefPair
+        }
+        val cefClient = JBCefApp.getInstance().createClient()
+        cefClient.setProperty("JS_QUERY_POOL_SIZE", 1)
+        val jbCefBrowser =
+            JBCefBrowserBuilder().setClient(cefClient).setEnableOpenDevToolsMenuItem(false)
+                .setMouseWheelEventEnable(true).build()
+        jbCefBrowser.setOpenLinksInExternalBrowser(true)
+        val jbCefPair = Pair(cefClient, jbCefBrowser)
+        return jbCefPair
+    }
```

Comment:
initialize the browser component only once, this prevents flickering and is more performant.